tests/manual-test-cases/Group5-Functional-Tests/5-22-NFS-Volume.robot

```diff
@@ -0,0 +1,166 @@
+    Set Global Variable  @{list}  ${esx3}  ${esx4}  ${esx5}  ${vc}  ${nfs_default_name}
+
+    Set Suite Variable  ${NFS_DEFAULT_IP}  ${nfs_default_ip}
+    Set Suite Variable  ${NFS_DEFAULT_NAME}  ${nfs_default_name}
+
+    #Need to set nfs volume at vic creation time
+    Install VIC Appliance To Test Server  additional-args=--volume-store="nfs://${NFS_DEFAULT_IP}/store?uid=0&gid=0:nfsVolumeStoreDefault"
+
+Verify NFS Volume Basic Setup
+    [Arguments]  ${prevOutput}  ${ContainerName}  ${nfsIP}  ${rwORro}
+
+    ${rc}  ${outputTemp}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run --name ${ContainerName} -d -v ${prevOutput}:/mydata ${busybox} mount
+    Should Be Equal As Integers  ${rc}  0
+
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} start -a ${ContainerName}
+    Should Be Equal As Integers  ${rc}  0
+    Should Contain  ${output}  ${nfsIP}://store/volumes/${prevOutput}
+    Should Contain  ${output}  /mydata type nfs (${rwORro}
+
+    ${ContainerRC}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} wait ${ContainerName}
+    Should Be Equal As Integers  ${ContainerRC}  0
+    Should Not Contain  ${output}  Error response from daemon
+
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} rm ${outputTemp}
+    Should Be Equal As Integers  ${rc}  0
+
+Verify NFS Volume Already Created
+    [Arguments]  ${containerVolName}
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} volume create --name=${containerVolName} --opt VolumeStore=nfsVolumeStoreDefault
+    Should Be Equal As Integers  ${rc}  1
+    Should Contain  ${output}  Error response from daemon: A volume named ${containerVolName} already exists. Choose a different volume name.
+
+
+*** Test Cases ***
+Simple docker volume create
+    Pull image  ${busybox}
+
+    ${rc}  ${outputDefault}=  Run And Return Rc And Output  docker %{VCH-PARAMS} volume create --opt VolumeStore=nfsVolumeStoreDefault
+    Should Be Equal As Integers  ${rc}  0
+
+    Set Suite Variable  ${UnnamedNFSVolContainer}  unnamednfsVolContainer
+    Set Suite Variable  ${nfsDefaultVolume}  ${outputDefault}
+
+    Verify NFS Volume Basic Setup  ${nfsDefaultVolume}  ${UnnamedNFSVolContainer}  ${NFS_DEFAULT_IP}  rw
+
+Docker volume create named volume
+    ${rc}  ${outputDefault}=  Run And Return Rc And Output  docker %{VCH-PARAMS} volume create --name nfs_default_%{VCH-NAME} --opt VolumeStore=nfsVolumeStoreDefault
+    Should Be Equal As Integers  ${rc}  0
+    Should Be Equal As Strings  ${outputDefault}  nfs_default_%{VCH-NAME}
+
+    Set Suite Variable  ${namedNFSVolContainer}  namednfsVolContainer
+    Set Suite Variable  ${nfsNamedVolume}  ${outputDefault}
+
+    Verify NFS Volume Basic Setup  ${nfsNamedVolume}  ${namedNFSVolContainer}  ${NFS_DEFAULT_IP}  rw
+
+Docker volume create already named volume
+    Verify NFS Volume Already Created  ${nfsDefaultVolume}
+
+    Verify NFS Volume Already Created  ${nfsNamedVolume}
```

Comment:
Good catch.  I'll get rid of the dependency.

I didn't want to repeat the same thing twice here, that's why I created the keyword but I can get rid of the keyword and just have the steps to verify that spelled out here instead.  I do this for the default and named nfs volume.  
@mhagen-vmware Want me to get rid of the keyword and just write out the commands here instead?